### PR TITLE
Pool statistics from the graph

### LINKED ISSUE #284: query subgraph for all pool statistics (part of TheGraph work)
Refactor to get all pool statistics from our subgraph, with Apollo's `useQuery` hook https://www.apollographql.com/docs/react/data/queries/

### PR DESCRIPTION
**Pull request by mihaisc**
Get pool statistics directly from the graph without connecting a wallet.

closes #284

### CODE REVIEW

File: src/utils/getAddressesForMain.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { ConfigHelper } from '@oceanprotocol/lib'
```
Comment (kremalicious) [inline src/utils/getAddressesForMain.ts:1]: do we need this utility?

**Comment by kremalicious** [comment]
This works amazing! 

Only have this one issue with the price graph where on first render it's fine but when clicking on another tab, and then back to Pool tab they render cut off for me:

Screen Shot 2020-12-10 at 20 06 20

Something definitely changed in rendering those, we also had a subtle animation when they were initially rendered.

**Comment by kremalicious** [comment]
But all data seems to be alright, also for the graphs when comparing live with this PR, the shapes very closely match with bigger differences only in the very beginning as it seems

Live:
Screen Shot 2020-12-10 at 20 16 31

The Graph:
Screen Shot 2020-12-10 at 20 16 24

**Comment by kremalicious** [comment]
more findings: we need to put the `ApolloProvider` and its `client` in the highest component possible. Otherwise we always have to create a new client in different places. Cause I also will need it for metadata history. And we need it for pool transactions. So we should maybe have some `getApolloClient()` helper passing `subgraphUri` to it

Could make sense adding the `ApolloProvider` the standard gatsby way with `wrapRootElement` but only in gatsby-browser.js but then again we have this problem with switching config

File: src/utils/getAddressesForMain.ts (hunk 1 above)
Comment (mihaisc) [reply src/utils/getAddressesForMain.ts:1]: no, i needed it before the last update in the graph, i had to filter the tokens, now the ocean reserve is on the transaction. Will remove it

**Comment by mihaisc** [comment]
You are right, i will move it somewhere under OceanProvider so i can catch config changes
I removed the animations because i with them the update looks bad, because i get chunks of 1k items
I'll check the bug when you switch tabs.

**Comment by kremalicious** [comment]
all the graph (the visual one) rendering issues seem to be gone, at least in Chrome and Safari. Might be some chart.js update which fixed it

**Review by kremalicious — APPROVED** [review]
working great! Can be released once Rinkeby & Ropsten subgraphs are up again

**Comment by kremalicious** [comment]
all confirmed working in all supported networks, wohoo

**Pull request merged**